IRremoteInt.h

```diff
@@ -211,6 +211,7 @@ typedef struct {
   unsigned int timer;     // state timer, counts 50uS ticks.
   unsigned int rawbuf[RAWBUF]; // raw data
   uint8_t rawlen;         // counter of entries in rawbuf
+  uint8_t overflow;
```

Comment:
reading other libraries and esp core I see how they most use *_t types and return normal types so sketches can use the normal types. we should probably continue the same. if it's internal in the lib keep using *_t and if it's going to be used in the user sketch then use normal types like int/unsigned int/bool/ etc..